---
float32[6] statusvalue